### ISSUE TITLE
Grave dissapears when killed by a dragon from "Ice and fire" mod

### ISSUE THREAD

**Issue by Sheppard22** [Corail31/tombstone_lite#15]
Hi, it happened to me many times now. When fire dragon kills me with fire, my grave dessapears, don't know why (Not sure about other dragons, i guess with them too)

**Comment by Corail31** [comment]
Hi Sheppard22,
I don't know if it's a mod incompatibility, but vanilla dragon can pass through.
But as i'm not a real player... it's hard for me to test these end game fights.
So, all infos will be welcome ;)

**Comment by Sheppard22** [comment]
Sorry, missing info. I'm playing on Minecraft 1.12.2 with tombstone-2.1.5-1.12 and Forge 14.23.1.2584. I think the grave is destroyed, because the key gets missing too. The dragon is from "iceandfire-1.3.1-1.12.2"

**Comment by Sheppard22** [comment]
The dragons from the mod spawn on the Overworld

**Comment by Corail31** [comment]
It's the good time for me to test this mod :)
But i think i prevented such errors in 2.2.0.

**Comment by Sheppard22** [comment]
I'll try with release 2.2.0, thanks! I'll share my results in some hours

**Comment by Sheppard22** [comment]
Hi! I got myself killed by a dragon and the bug didn't show up, so maybe it's fixed, tomorrow i'll test it further to see if the bug is gone. btw, nice mod! :D

**Comment by Corail31** [comment]
Happy to heard it, i also tested "fire & ice" without trouble, and i liked the dragons :p (i think it the trouble was linked to fire) and thanks for the report :+1:

**Comment by Sheppard22** [comment]
Hi again! i don't really know if i must create a new issue or just writte here even though it's labeled as "closed", but the bug is still there.

 I think i know what happens (This time a Green Dragon killed me with fire) When this dragons throws fire, blocks affected by it turns into "Charred Stone" (A block from ice & fire). I have my death's waypoint inside this block so i think it makes sense that the grave has been replaced by Charred Stone. 

I took some screenshots:

![2018-01-08_15 41 37](https://user-images.githubusercontent.com/34324441/34687282-fe0c7736-f48c-11e7-976a-da0531d0201f.png)

![2018-01-08_15 42 44](https://user-images.githubusercontent.com/34324441/34687252-e99d9532-f48c-11e7-87c9-6d958807be60.png)

Happened to her too:
![2018-01-08_15 43 51](https://user-images.githubusercontent.com/34324441/34687526-b4eebe5a-f48d-11e7-9099-556f168c1992.png)
